README.md

```diff
@@ -67,19 +67,33 @@ Check them using command like:
 (zcat /proc/config.gz || cat /boot/{config,config-$(uname -r)}) | grep 'CONFIG_DEBUG_INFO_BTF='
 ```
 
+### Enable IP Forwarding
+
+By default, any latest Linux distributions will have IP Forwarding `disabled`. In the case where we need to up a Linux router/gateway or a VPN server or simply a plain dial-in server, then we must need to enable forwarding. Do the followings to have `ip-forwarding` feature enabled.
+
+```shell
+cat >>/etc/sysctl.d/dae.conf<<EOF
```

Comment:
User may need `sudo` here.

Reply:
Resolved in [commit](https://github.com/v2rayA/dae/pull/15/commits/1d2aa7d526bf65758735c873a90c0e4fd6202248)

Reply:
You cannot use `sudo` to redirect output. You can try.

Reply:
We can use:
```
sudo tee /etc/sysctl.d/dae.conf << 'EOF'
net.ipv4.ip_forward = 1
net.ipv6.conf.all.forwarding = 1
EOF
```

Reply:
Sure. Let's use the above fix.